libethereum/State.cpp

```diff
@@ -186,11 +192,13 @@ Account* State::account(Address const& _addr)
 	clearCacheIfTooLarge();
 
 	RLP state(stateBack);
-	auto i = m_cache.emplace(
-		std::piecewise_construct,
-		std::forward_as_tuple(_addr),
-		std::forward_as_tuple(state[0].toInt<u256>(), state[1].toInt<u256>(), state[2].toHash<h256>(), state[3].toHash<h256>(), Account::Unchanged)
-	);
+	auto i = m_cache.emplace(std::piecewise_construct,
+							 std::forward_as_tuple(_addr),
+							 std::forward_as_tuple(state[0].toInt<u256>(),
+												   state[1].toInt<u256>(),
```

Comment:
This one is also weird because of `TabWidth: 4` I think

Reply:
Yes.